### PR TITLE
Migrate pension and burials form to new zip code format

### PR DESCRIPTION
**Pull request by jbalboni**
Adds new validation and migrations for ICMHS zip code handling, since there are some restrictions on the format for US addresses.

screen shot 2018-06-04 at 9 43 46 am

### CODE REVIEW

File: src/applications/common/schemaform/definitions/address.js [hunk 1]
```diff
@@ -73,8 +73,9 @@ const requiredFields = ['street', 'city', 'country', 'state', 'postalCode'];
  * @param {object} schema - Schema for a full form, including address definition in definitions
  * @param {boolean} isRequired - If the address is required or not, defaults to false
```
Comment (rianfowler) [inline src/applications/common/schemaform/definitions/address.js]: what is `addressProperty`

File: src/applications/pensions/config/form.js [hunk 2]
```diff
@@ -1544,7 +1548,11 @@ const formConfig = {
           path: 'additional-information/contact',
           uiSchema: {
             'ui:title': 'Contact information',
-            veteranAddress: address.uiSchema('Mailing address'),
+            veteranAddress: _.set(
+              'ui:validations[1]',
```
Comment (rianfowler) [inline src/applications/pensions/config/form.js]: How is this used?

**Review by rianfowler — APPROVED** [review]

File: src/applications/pensions/config/form.js (hunk 2 above)
Comment (jbalboni) [reply src/applications/pensions/config/form.js]: This is adding the central mail zip code validation to the list of validation functions, which already has one function for the general address validation.

File: src/applications/common/schemaform/definitions/address.js (hunk 1 above)
Comment (jbalboni) [reply src/applications/common/schemaform/definitions/address.js]: Added an additional JsDoc comment below.

**Review by lbai001 — APPROVED** [review]

**Pull request merged**